includes/reader-revenue/my-account/class-woocommerce-my-account.php

```diff
@@ -333,11 +339,13 @@ public static function is_user_verified() {
 	 * Redirect to "Account details" if accessing "My Account" directly.
 	 * Do not redirect if the request is a resubscribe request, as resubscribe
 	 * requests do their own redirect to the cart/checkout page.
+	 * Do not redirect if this request is a membership cancellation.
 	 */
 	public static function redirect_to_account_details() {
-		$resubscribe_request = isset( $_REQUEST['resubscribe'] ) ? 'shop_subscription' === get_post_type( absint( $_REQUEST['resubscribe'] ) ) : false; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$resubscribe_request       = isset( $_REQUEST['resubscribe'] ) ? 'shop_subscription' === get_post_type( absint( $_REQUEST['resubscribe'] ) ) : false; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
```suggestion
		$resubscribe_request       = isset( $_REQUEST['resubscribe'] ) ? 'shop_subscription' === \get_post_type( absint( $_REQUEST['resubscribe'] ) ) : false; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

This already wasn't globally namespaced in `release`, but since we're changing this line anyway we may as well add it!